source/production/F0.Minesweeper.Components/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,19 @@
+using F0.Minesweeper.Components.Abstractions;
+using F0.Minesweeper.Components.Logic.Cell;
+using F0.Minesweeper.Components.Logic.Game;
+using Microsoft.Extensions.DependencyInjection;
+using Prism.Events;
+
+namespace F0.Minesweeper.Components.Extensions
+{
+	public static class IServiceCollectionExtensions
+	{
+		public static IServiceCollection AddMinesweeperComponentsServices(this IServiceCollection serviceCollection)
+		{
+			serviceCollection.AddSingleton<IEventAggregator>(new EventAggregator());
```

Comment:
I usually use the ``Func`2`` overload over the "instance"-overload, so that the DI container can manage the lifetime of the instance.
My question now is: is `EventAggregator` _disposable_?

Reply:
As discussed switched to just use the generic version since we have a parameterless constructor